syft/frameworks/torch/tensors/interpreters/native.py

```diff
@@ -929,17 +929,23 @@ def share(
             if self.type() == "torch.FloatTensor":
                 raise TypeError("FloatTensor cannot be additively shared, Use fix_precision.")
 
-            shared_tensor = (
-                syft.AdditiveSharingTensor(
-                    protocol=protocol,
-                    field=field,
-                    dtype=dtype,
-                    crypto_provider=crypto_provider,
-                    owner=self.owner,
-                )
-                .on(self.copy(), wrap=False)
-                .share_secret(*owners)
-            )
+            else:
```

Comment:
I think you can remove the else here - if it enters the throw

Reply:
done :)